### PR TITLE
Assignments

### PR DESCRIPTION
**Pull request by marten**

### CODE REVIEW

File: app/operations/assignments/create.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+module Assignments
+  class Create < Operation
+    integer :base_workflow_id, default: Rails.application.secrets["zooniverse_project_id"]
```
Comment (camallen) [inline app/operations/assignments/create.rb]: any reason for the naming mismatch, base workflow vs project id?

File: config/routes.rb [hunk 2]
```diff
@@ -4,10 +4,13 @@
 
   resources :users
 
+  resources :assignments
+
   namespace :teachers do
     resources :classrooms do
       resources :student_users
     end
+
```
Comment (camallen) [inline config/routes.rb]: newline?

**Comment by camallen** [comment]
LGTM - some minor comments

File: app/operations/assignments/create.rb (hunk 1 above)
Comment (marten) [inline app/operations/assignments/create.rb]: That's me screwing up a copy/paste. Nice catch.

**Pull request merged**